tests/text_quotations_test.py

```diff
@@ -807,7 +811,7 @@ def test_split_email():
         >
         >
 """
-    expected_markers = "stttttsttttetesetesmmmmmmssmmmmmmsmmmmmmmm"
+    expected_markers = "stttttsttttetesetesmmmmmmsmmmmmmmmmmmmmmmm"
```

Comment:
Hi @obukhov-sergey Hope you can help me :-)

I recently upgraded my Talon version from a version before this change to a version after, and some of my tests started failing. 

I am just trying to understand this change. Looking at this test and its expectation:

![image](https://user-images.githubusercontent.com/55391479/197205522-21a45de9-90a7-4407-a187-2b893d5c3863.png)

these lines:

```
> > Date: Mon, 2 Apr 2012 17:44:22 +0400
```

```
> > > From: bob@xxx.mailgun.org
```

were previously classified as splitter lines ('s'), but are now classified as lines starting with quotation marker '>' ('m').

Was the previous behavior (classification as splitter lines) incorrect?

I am trying to figure out if I was previously relying on buggy behavior that has now been fixed?  (And if so, I should update my tests). 

Many thanks